fms/modules/feedforward.py

```diff
@@ -45,15 +45,17 @@ def __init__(
         gain=1,
     ):
         super(FeedForwardBlock, self).__init__()
-        hidden_dim = int(hidden_grow_factor * emb_dim)
+        self.hidden_dim = int(hidden_grow_factor * emb_dim)
```

Comment:
why making this a member variable? doesn't look like it's used anywhere outside this constructor

Reply:
its being used in the `TPFeedForwardBlock`, just hadn't been called as the other models use GatedLinearUnit